ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayLayerViewDrawStatus/DisplayLayerViewDrawStatus.h

```diff
@@ -33,13 +33,11 @@ namespace Esri
 
 #include <QMetaObject>
 #include <QQuickItem>
-#include <QStringList>
 
 class DisplayLayerViewDrawStatus : public QQuickItem
 {
   Q_OBJECT
-  Q_PROPERTY(QStringList layerNames READ layerNames NOTIFY namesChanged)
-  Q_PROPERTY(QStringList layerViewStates READ layerViewStates NOTIFY statusChanged)
+  Q_PROPERTY(const QList<QObject*>& statusModel READ statusModel NOTIFY modelChanged)
```

Comment:
@anmacdonald is this safe? I'm sure it's working, but some type conversions happen when converting lists between C++ and Qml and I don't know if this is recommended best practices.

I'm not sure this type of conversion is officially supported.
http://doc.qt.io/qt-5/qtqml-cppintegration-data.html#sequence-type-to-javascript-array

Reply:
Ok, that makes more sense. You're providing as a model and that's what the Qt example shows. All good 👍